spring-security/src/main/java/com/epam/esm/user/authComponent/AuthUserComponent.java

```diff
@@ -0,0 +1,22 @@
+package com.epam.esm.user.authComponent;
+
+import com.epam.esm.exceptionhandler.exception.ServerException;
+import com.epam.esm.role.Role;
+import com.epam.esm.user.model.User;
+import com.epam.esm.utils.validation.DataValidation;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+@Component
+@Slf4j
+public class AuthUserComponent {
+    public boolean hasPermission(User user, Long id) {
```

Comment:
I don't really like name Component here. Please rename to something more informative

Reply:
replaced with  AuthOrderVerify